src/components/ReplayableVideoAsset.vue

```diff
@@ -92,6 +103,7 @@ export default {
       if (this.videoEnded) return this.$t('video.replay');
       return this.isPlaying ? this.$t('video.pause') : this.$t('video.play');
     },
+    ariaLabelledByContainer: ({ id }) => `custom-controls ${id}-alt`,
```

Comment:
Which element would this be referring to? I don't see it being set as the ID of any element, and I don't think this would even be a valid ID value since it has whitespace.

Reply:
The `aria-labelledby` property takes as value a space-separated id reference list, which means you can combine more than one element into a single accessible name. It will read the content of `custom-controls` id first and then the id alt text.

Source: https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby

Reply:
Thanks, that makes sense.

We may need to make the 2nd part conditional since the element with the ID for the alt text will only exist if there was alt text provided.